crates/turborepo-lib/src/commands/mod.rs

```diff
@@ -125,15 +125,27 @@ impl CommandBase {
         let team_id = config.team_id();
         let team_slug = config.team_slug();
 
-        let Some(token) = config.token() else {
-            return Ok(None);
+        let auth_file_path = self.global_auth_path()?;
+        let config_file_path = self.global_config_path()?;
+        let client = self.api_client()?;
+        let auth = match turborepo_auth::read_auth_file(&auth_file_path) {
+            Ok(auth) => auth,
+            Err(_) => match turborepo_auth::read_auth_file(&config_file_path) {
```

Comment:
I think this always fails. We're trying to deserialize the old `config.json` as an `auth.json` and it'll be missing the `tokens` field.

Reply:
Yeah, I can just use the existing read function now that it's sync. I'll clean up.